tasks/internal_reconcile_report_phantom/test/unit_tests/test_internal_reconcile_report_phantom.py

```diff
@@ -0,0 +1,25 @@
+"""
+Name: test_internal_reconcile_report_phantom.py
+Description:  Unit tests for internal_reconcile_report_phantom.py.
+"""
+import datetime
+import json
+import random
+import unittest
+import uuid
+from unittest.mock import Mock, call, patch, MagicMock
+
+from fastjsonschema import JsonSchemaValueException
+
+import internal_reconcile_report_phantom
+from orca_shared.reconciliation import shared_reconciliation
+
+
+class TestInternalReconcileReportPhantom(
+    unittest.TestCase
+):  # pylint: disable-msg=too-many-instance-attributes
+    """
+    TestInternalReconcileReportPhantom.
+    """
+def test_dummy():
```

Comment:
Spacing is wrong here, should have four spaces so it is under the class.

Reply:
oops fixed